pmd-apex/pom.xml

```diff
@@ -54,6 +54,13 @@
             <type>pom</type>
         </dependency>
 
+        <dependency>
+            <groupId>com.github.nawforce</groupId>
+            <artifactId>apexlink</artifactId>
+            <version>1.1.1</version>
```

Comment:
Note: there is already 1.2.2 available: https://repo1.maven.org/maven2/com/github/nawforce/apexlink/